src/app/games/martian-chess/MartianChessMove.ts

```diff
@@ -51,7 +51,7 @@ export class MartianChessMove extends MoveCoordToCoord {
                                       this.getEnd().x + ', ' + this.getEnd().y + ')' +
                                       ending + ')';
     }
-    public equals(other: MartianChessMove): boolean {
+    public override equals(other: MartianChessMove): boolean {
         if (this.getStart().equals(other.getStart()) === false) return false;
         if (this.getEnd().equals(other.getEnd()) === false) return false;
         return this.calledTheClock === other.calledTheClock;
```

Comment:
return super.equals() && this.calledTheClock === other.calledTheClock?

Reply:
c'est quoi l'erreur de compil ? Car contourner l'héritage c'pas ouf

Reply:
public override equals(other: MartianChessMove): boolean {
        if (super.equals(other as this) === false) return false;
        return this.calledTheClock === other.calledTheClock;
    }
    Cébon lo !